TF_Tooling_Web/src/TouchFree.ts

```diff
@@ -0,0 +1,29 @@
+import { ConnectionManager } from "./Connection/ConnectionManager";
+import { SVGCursor, SVGCursorParams } from "./Cursors/SvgCursor";
+import { TouchlessCursor } from "./Cursors/TouchlessCursor";
+import { WebInputController } from "./InputControllers/WebInputController";
+
+var InputController: WebInputController;
+export var CurrentCursor: TouchlessCursor;
+
+export interface TfInitParams {
+    initialiseCursor?: boolean;
+}
+
+export function Init(_tfInitParams?: TfInitParams ): void {
+    ConnectionManager.init();
+
+    ConnectionManager.AddConnectionListener(() => {
+        InputController = new WebInputController();
+
+        if (_tfInitParams === undefined) {
+            CurrentCursor = new SVGCursor();
+        } else {
+            if (_tfInitParams.initialiseCursor === undefined) {
```

Comment:
Could combine this into one check?

Reply:
This if with the one above? If we do that the else if following this if would be messy because of not knowing what's there. I think you'd need to basically do this:

```
if (_tfInitParams === undefined || _tfInitParams.initialiseCursor === undefined) {
    CurrentCursor = new SVGCursor();
} else if (_tfInitParams !== undefined && _tfInitParams !== undefined && _tfInitParams.initialiseCursor) {
    CurrentCursor = new SVGCursor();
}
```

Reply:
No I meant with the one after it (initialiseCursor and true check) - that is a monstrosity ^